src/Rakaposhi.Core/Rakaposhi.Business.Core/DBRepository/DBRepositoryFactory.cs

```diff
@@ -6,12 +6,13 @@ public class DBRepositoryFactory : IRepositoryFactory
     {
         private IUserRoleRepository _userRoleRepository;
         private IUserStatusRepository _userStatusRepository;
+        private ITransTypeRepository _transTypeRepository;
 
         public IUserRoleRepository UserRoleRepository
         {
             get
             {
-                if(_userRoleRepository is null)
+                if (_userRoleRepository is null)
```

Comment:
What I feel, It is not a change of this PR.
Please Remove space between if and its brackets.

Reply:
fixed